packages/plugin-xml/src/Attributes.ts

```diff
@@ -0,0 +1,45 @@
+import { Modifier } from '../../core/src/Modifier';
+import { deepEqualObjects } from '../../utils/src/utils';
+
+export class Attributes extends Modifier {
+    value: Record<string, string> = {};
```

Comment:
I really have an issue with this `value`. I understand that `Attributes` cannot have the keys on itself and that is why it exists. However, I would make this `private` and call it differently. Maybe `record` ? `cache` ? Sounds complicated. Do you have another idea ?

Reply:
Yes I dislike it too. However `value` is the clearest name I can come up with... I do like the idea of making it private so I made it private, renamed it `_value` and added two methods: `keys` and `values` that work like `Map.keys()` and `Map.values()`, respectively.

Reply:
When I think `value` I think `42` or `coucou`, not `an entire record of multiple key/value pair`. What is really confusing is that to *describe* what `value` is in the previous sentence, I had to use the word "value" in "key/value pair", so `value` is like the most confusing name for me... :p

Reply:
`_record` then?

Reply:
Yup, unless you have another name in mind which do not have the same problem as `value`.

Reply:
I don't.